pihole/datadog_checks/pihole/pihole.py

```diff
@@ -2,8 +2,8 @@
 
 
 class PiholeCheck(AgentCheck):
-    def __init__(self, name, init_config, instance):
-        super(PiholeCheck, self).__init__(name, init_config, instance)
+    def __init__(self, name, init_config, instances=None):
```

Comment:
```suggestion
    def __init__(self, name, init_config, instances):
```
`instances` cannot be `None`